python/pen_checker_cdc.py

```diff
@@ -242,7 +264,12 @@ def process_pbp(isolate,protein_fasta,tpd_start,tpd_end,tpd_lab,results_csv,k):
     input_gene_names.append(gene_name.lower())
     all_gene_names = set(input_gene_names)
     gene_length = files_for_input.tlength
-    
+
+    if files_for_input.pbp not in ['pbp1a','pbp2b','pbp2x']:
+        aa_dir_name = "./" + re.sub(".csv","",files_for_input.output) + "_aa_dir"
+        os.mkdir(aa_dir_name)
+        all_gene_names = [files_for_input.pbp]
```

Comment:
Making new dir for aa sequences, have to remember to delete for every new run if it doesn't run all the way through